specification/devcenter/DevCenter/service.tsp

```diff
@@ -36,7 +33,11 @@ namespace DevCenterService;
 
 @doc("DevCenter API versions")
 enum APIVersions {
-  @doc("The 2023-04-01 service API version")
+  // @doc("The 2023-04-01 service API version")
+  // @useDependency(Versions.v1_0_Preview_2)
+  // v2023_04_01: "2023-04-01",
```

Comment:
This api version needs to remain in the spec. Please uncomment it

Reply:
Undone. There were temp changes to make reviewing the breaking changes a bit easier, it's back to normal now